src/main/java/cz/brmlab/yodaqa/flow/dashboard/QuestionDashboard.java

```diff
@@ -26,7 +26,7 @@ private QuestionDashboard() {}
 	public static QuestionDashboard getInstance() {
 		return qd;
 	}
-
+	private Map<Integer, AnsweringSnippet> snippets = new HashMap<>();
```

Comment:
Yes, per question list of snippets is better, will fix it at around 2am. This was from when we used a global list of Passages (strings).